lib/components/narrative/narrative-itineraries-header.js

```diff
@@ -72,15 +59,31 @@ export default function NarrativeItinerariesHeader ({
         : <>
           <div
             style={{flexGrow: 1}}
-            title={titleText}
+            title={intl.formatMessage({
+              id: 'components.NarrativeItinerariesHeader.titleText',
+              values: {
+                issueNum: errors.length,
+                itineraryNum: itineraries.length,
+                pending: pending ? 'true' : 'false'
+              }
+            })}
           >
             <span style={{marginRight: '10px'}}>
-              {resultText}
+              <FormattedMessage
+                id='components.NarrativeItinerariesHeader.resultText'
+                values={{
+                  itineraryNum: itineraries.length,
+                  pending: pending ? 'true' : 'false'
+                }}
+              />
             </span>
             {errors.length > 0 && (
               <IssueButton onClick={onToggleShowErrors}>
                 <Icon style={{fontSize: 11, marginRight: 2}} type='warning' />
-                <span>{errors.length} issues</span>
+                <span><FormattedMessage
+                  id='components.NarrativeItinerariesHeader.numIssues'
+                  values={{issueNum: errors.length}}
+                /></span>
```

Comment:
Suggested formatting:

```suggestion
                <span>
                  <FormattedMessage
                    id='components.NarrativeItinerariesHeader.numIssues'
                    values={{issueNum: errors.length}}
                  />
                </span>
```

Reply:
adjusted in https://github.com/opentripplanner/otp-react-redux/pull/430/commits/ad8ea3326848a5ac077e01898d7f7657412fda84